tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -11,6 +11,7 @@ Standard output:
 ../../../../built/local/lib.dom.d.ts(11899,13): error TS2300: Duplicate identifier 'Request'.
 ../../../../built/local/lib.dom.d.ts(16316,11): error TS2300: Duplicate identifier 'Window'.
 ../../../../built/local/lib.dom.d.ts(16447,13): error TS2300: Duplicate identifier 'Window'.
+../../../../built/local/lib.dom.d.ts(17190,15): error TS2451: Cannot redeclare block-scoped variable 'name'.
```

Comment:
Conflict with a typedef *property* `name` that we incorrectly parse as a typedef `name`.